packages/terra-form-input/tests/nightwatch/FormInputPropPermutations.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import FormInput from '../../lib/Input';
```

Comment:
This file will need changes with updates in packages/terra-form-input/src/Input.jsx for exporting the component as Input instead of FormInput